### PR TITLE
Fixing URL in the statistics template.

### PR DESCRIPTION
**Pull request by kongtiaowang**

### CODE REVIEW

**Comment by kongtiaowang** [comment]
Remove main.php in the .tpl files.

**Comment by johnsaigle** [comment]
Decision was to add the LORIS base URL (pretty-form) so that we're not making assumptions about the URL.

**Comment by ridz1208** [comment]
need to use absolute urls

**Comment by johnsaigle** [comment]
Is says there are conflicts. Maybe rebase?

**Comment by driusan** [comment]
I don't think these URLs will work, either. they're missing either the main.php or the pretty url path for the module after the baseurl.

File: modules/statistics/templates/menu_statistics_mri_site.tpl [hunk 1]
```diff
@@ -28,9 +28,9 @@
       {foreach from=$AllVisits item=visit name=VisitLoop}
       <td>{foreach from=$data[item].incompletes[$visit] item=Candidate name=CandLoop}
             {if $Candidate.test_url == "PF_Missing"}
-                <a href="main.php?test_name=imaging_browser&subtest=viewSession&sessionID={$Candidate.SessionID}">
+                <a href="{$baseurl}/statistics/?subtest=viewSession&sessionID={$Candidate.SessionID}">
```
Comment (driusan) [inline modules/statistics/templates/menu_statistics_mri_site.tpl]: this is the wrong url. It should be {$baseurl}/imaging_browser/viewSession/?...

**Comment by kongtiaowang** [comment]
I have fixed them. Thanks, it is time to pass them.

File: modules/statistics/templates/menu_statistics_dd_site.tpl [hunk 2]
```diff
@@ -18,7 +18,7 @@
       <td>{$data[item].count}</td>
       {foreach from=$AllVisits item=visit name=VisitLoop}
       <td>{foreach from=$data[item].incompletes[$visit] item=Candidate name=CandLoop}
-            <a href="main.php?test_name={$Candidate.test_url}&candID={$Candidate.CandID}&sessionID={$Candidate.SessionID}&commentID={$Candidate.CommentID}">{$Candidate.PSCID}</a>
+            <a href="{$baseurl}/{$Candidate.test_url}/?candID={$Candidate.CandID}&sessionID={$Candidate.SessionID}&commentID={$Candidate.CommentID}">{$Candidate.PSCID}</a>
```
Comment (driusan) [inline modules/statistics/templates/menu_statistics_dd_site.tpl]: I'm not sure if this is the most appropriate URL (I think baseURL/candID/sessionID/test_name/ might be cleaner), but this one should work too..

**Pull request merged**